services/manage/app/mixins/action-mixin.js

```diff
@@ -26,6 +26,14 @@ export default Mixin.create(LoadingMixin, {
     this.hideLoading();
   },
 
+  validateLoginLinkTemplateObj(loginLinkTemplate = {}){
```

Comment:
Is this necessary?

Reply:
Agreed, this should be removed. There shouldn't be a need to validate and modify the value here, so long as it is returned from the GraphQL query properly, and initialized on create so that the inputs can bind properly. Since this is being used on submission, that would seem to have already been accounted for.

Reply:
```js
formatLoginLinkTemplateInput({ subject, unverifiedVerbiage, verifiedVerbiage } = {}){
  return {
    subject,
    unverifiedVerbiage,
    verifiedVerbiage,
  };
},
```